auth-web/src/components/auth/common/SingleProduct.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <div>
+    <template>
+      <v-card
+        outlined
+        hover
+        class="product-card py-8 px-5 mb-4 elevation-1"
+        :class="[{'px-8': icon === ''}, {'processing-card' : !isRRequesting}]"
+        :data-test="`div-product-${productDetails.title}`"
+      >
+        <div>
+          <header class="d-flex align-center">
+            <div class="product-icon-container mt-n2 mr-2" v-if="!isRRequesting">
+              <v-icon meduim color="primary">{{icon}}</v-icon>
+            </div>
+            <div class="pr-8 ">
+              <h3 class="title font-weight-bold product-title mt-n1">{{productDetails.name}}</h3>
+              <div>{{productSubTitle}}</div>
+            </div>
+            <v-icon large v-if="isRequestNow"
+            class="ml-auto"
+            @click="requestNow()">mdi-close</v-icon>
+            <v-btn
+            v-else
+              large
+              depressed
+              color="primary"
+              width="120"
+              class="font-weight-bold ml-auto"
+              :outlined="!isApproved"
+              :aria-label="`Select  ${productDetails.name}`"
+              :data-test="`btn-productDetails-${productDetails.name}`"
+              @click="requestNow()"
+            >
+              <span>{{label}}</span>
+            </v-btn>
+          </header>
+
+          <div class="product-card-contents">
+            <v-expand-transition>
+              <div v-if="isRequestNow">
+                <div class="pt-7 mb-7" >
+                  <v-divider class="mb-7"></v-divider>
+                  <p class="mb-7">The search and registration products are intended for the exclusive use of solicitors and notaries only.
```

Comment:
Better to move this to a component, as all product selection  won't be having same terms or fields